From 933bd6cf43200fc525252a0310a0d4e8ba158a68 Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Fri, 24 Mar 2017 11:21:30 +0100
Subject: [PATCH 48/51] FM: Fix KHz->kHz

* Kilohertz == kHz
* Fix spacing issues while on it
* Remove redundant units from ranges

Change-Id: I8115969683c62a920af330ca5b591afd3a085b1e
---
 fmapp2/res/values/arrays.xml  |  6 +--
 fmapp2/res/values/strings.xml | 76 +++++++++++++++++------------------
 2 files changed, 41 insertions(+), 41 deletions(-)

diff --git a/fmapp2/res/values/arrays.xml b/fmapp2/res/values/arrays.xml
index 56a5524..fc60d74 100644
--- a/fmapp2/res/values/arrays.xml
+++ b/fmapp2/res/values/arrays.xml
@@ -328,9 +328,9 @@
   </string-array>
 
   <string-array name="channel_spacing_entries">
-      <item>50 KHz</item>
-      <item>100 KHz</item>
-      <item>200 KHz</item>
+      <item>50 kHz</item>
+      <item>100 kHz</item>
+      <item>200 kHz</item>
   </string-array>
 
 </resources>
diff --git a/fmapp2/res/values/strings.xml b/fmapp2/res/values/strings.xml
index 6caed6b..b37fe27 100644
--- a/fmapp2/res/values/strings.xml
+++ b/fmapp2/res/values/strings.xml
@@ -105,43 +105,43 @@
     <string name="regional_band_entry_united_states">United States</string>
     <string name="regional_band_entry_user_defined">User defined band</string>
 
-    <string name="regional_band_summary_north_america">North America (87.5MHz to 108.0MHz in 200 KHz steps)</string>
-    <string name="regional_band_summary_europe">Europe (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_japan">Japan (76.0MHz to  90.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_japan_wide">Japan (wide) (90.0MHz to 108.0MHz in 50 KHz steps)</string>
-    <string name="regional_band_summary_australia">Australia (87.7MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_austria">Austria (87.5MHz to 108.0MHz in  50 KHz steps)</string>
-    <string name="regional_band_summary_belgium">Belgium (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_brazil">Brazil (87.5MHz to 108.0MHz in 200 KHz steps)</string>
-    <string name="regional_band_summary_china">China (87.0MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_czech_republic">Czech Republic (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_denmark">Denmark (87.5MHz to 108.0MHz in 50 KHz steps)</string>
-    <string name="regional_band_summary_finland">Finland (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_france">France (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_germany">Germany (87.5MHz to 108.0MHz in 50 KHz steps)</string>
-    <string name="regional_band_summary_greece">Greece (87.5MHz to 108.0MHz in 50 KHz steps)</string>
-    <string name="regional_band_summary_hong_kong">Hong Kong (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_india">India (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_indonesia">Indonesia (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_ireland">Ireland (87.5MHz to 108.0MHz in 50 KHz steps)</string>
-    <string name="regional_band_summary_italy">Italy (87.5MHz to 108.0MHz in 50 KHz steps)</string>
-    <string name="regional_band_summary_korea">Korea (87.5MHz to 108.0MHz in 200 KHz steps)</string>
-    <string name="regional_band_summary_mexico">Mexico (88.1MHz to 107.9MHz in 200 KHz steps)</string>
-    <string name="regional_band_summary_netherlands">Netherlands (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_new_zealand">New Zealand (88.0MHz to 107.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_norway">Norway (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_poland">Poland (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_portugal">Portugal (87.5MHz to 108.0MHz in 50 KHz steps)</string>
-    <string name="regional_band_summary_russia">Russia (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_singapore">Singapore (88.0MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_slovakia">Slovakia (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_spain">Spain (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_switzerland">Switzerland (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_sweden">Sweden (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_taiwan">Taiwan (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_turkey">Turkey (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_united_kingdom">United Kingdom (87.5MHz to 108.0MHz in 100 KHz steps)</string>
-    <string name="regional_band_summary_united_states">United States (88.1MHz to 107.9MHz in 200 KHz steps)</string>
+    <string name="regional_band_summary_north_america">North America (87.5 to 108.0 MHz in 200 kHz steps)</string>
+    <string name="regional_band_summary_europe">Europe (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_japan">Japan (76.0 to 90.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_japan_wide">Japan (wide) (90.0 to 108.0 MHz in 50 kHz steps)</string>
+    <string name="regional_band_summary_australia">Australia (87.7 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_austria">Austria (87.5 to 108.0 MHz in 50 kHz steps)</string>
+    <string name="regional_band_summary_belgium">Belgium (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_brazil">Brazil (87.5 to 108.0 MHz in 200 kHz steps)</string>
+    <string name="regional_band_summary_china">China (87.0 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_czech_republic">Czech Republic (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_denmark">Denmark (87.5 to 108.0 MHz in 50 kHz steps)</string>
+    <string name="regional_band_summary_finland">Finland (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_france">France (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_germany">Germany (87.5 to 108.0 MHz in 50 kHz steps)</string>
+    <string name="regional_band_summary_greece">Greece (87.5 to 108.0 MHz in 50 kHz steps)</string>
+    <string name="regional_band_summary_hong_kong">Hong Kong (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_india">India (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_indonesia">Indonesia (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_ireland">Ireland (87.5 to 108.0 MHz in 50 kHz steps)</string>
+    <string name="regional_band_summary_italy">Italy (87.5 to 108.0 MHz in 50 kHz steps)</string>
+    <string name="regional_band_summary_korea">Korea (87.5 to 108.0 MHz in 200 kHz steps)</string>
+    <string name="regional_band_summary_mexico">Mexico (88.1 to 107.9 MHz in 200 kHz steps)</string>
+    <string name="regional_band_summary_netherlands">Netherlands (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_new_zealand">New Zealand (88.0 to 107.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_norway">Norway (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_poland">Poland (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_portugal">Portugal (87.5 to 108.0 MHz in 50 kHz steps)</string>
+    <string name="regional_band_summary_russia">Russia (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_singapore">Singapore (88.0 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_slovakia">Slovakia (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_spain">Spain (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_switzerland">Switzerland (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_sweden">Sweden (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_taiwan">Taiwan (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_turkey">Turkey (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_united_kingdom">United Kingdom (87.5 to 108.0 MHz in 100 kHz steps)</string>
+    <string name="regional_band_summary_united_states">United States (88.1 to 107.9 MHz in 200 kHz steps)</string>
     <string name="regional_band_summary_user_defined">User defined band</string>
 
     <string name="aud_output_mode">Audio output mode</string>
@@ -217,7 +217,7 @@
     <string name="chanl_spacing">Channel spacing</string>
 
     <string name="set">Set</string>
-    <string name="user_defind_band_msg">Enter frequency from 76.0 to 108.0 MHz with minimum 1 channel spacing and 100 KHz space between min and max frequency</string>
+    <string name="user_defind_band_msg">Enter frequency from 76.0 to 108.0 MHz with minimum 1 channel spacing and 100 kHz space between min and max frequency</string>
     <string name="save_record_file">FM recorded file saved to \'<xliff:g id="record_file">%1$s</xliff:g>\'</string>
     <string name="fm_search_no_results">No compatible stations found</string>
 
-- 
2.17.1

